js.go

```diff
@@ -1489,6 +1500,14 @@ func BindStream(name string) SubOpt {
 	})
 }
 
+// BindConsumer binds a subscription to a consumer based on a durable name.
+func BindConsumer() SubOpt {
```

Comment:
By default this happens sometimes on its own if the consumer exists. So possibly the naming might be a bit off here.

BindOnly or NoCreate. Probably not these but hopefully this gets to the point I am trying to make a bit.

Reply:
Updated to use `BindOnly` for now